README.md

```diff
@@ -112,10 +112,11 @@ $root = $tracer->startActiveSpan('php');
 
         $http = $tracer->startActiveSpan('http');
         file_get_contents('http://php.net');
```

Comment:
I think this example is promoting bad naming practice. `root` and `http` should really be called `scope`, and they should be in different syntax scopes (not sure if php allows that - if not, best to introduce a function in the example)